paderbox/array/interval/core.py

```diff
@@ -46,6 +46,27 @@ def intervals_to_str(intervals):
     return ', '.join(f'{start}:{end}' for start, end in intervals)
 
 
+def _validate_shape(shape):
+    if shape is None or isinstance(shape, int):
+        return
+    if not isinstance(shape, (tuple, list)):
+        raise TypeError(f'Invalid shape {shape} of type {type(shape)}')
+    if not len(shape) == 1:
+        raise ValueError(f'Invalid shape {shape} has to have length 1')
```

Comment:
Can you add a comment to the source code?
Something like:
`As of now, we only support 1D. We haven't decided, how to handle what higher shapes mean. Probably the last will be, what it is now and the earlier dims will be independent dims.`